tests/components/zwave_js/conftest.py

```diff
@@ -10,17 +10,15 @@
 from zwave_js_server.model.node import Node
 from zwave_js_server.version import VersionInfo
 
-from homeassistant.helpers.device_registry import (
-    async_get_registry as async_get_device_registry,
-)
+from homeassistant.helpers.device_registry import async_get as async_get_device_registry
 
 from tests.common import MockConfigEntry, load_fixture
 
 
 @pytest.fixture(name="device_registry")
 async def device_registry_fixture(hass):
```

Comment:
What a weird fixture.

Reply:
🤷🏾‍♂️ I can get rid of it if we want, I didn't get it either but didn't want to increase the scope of the refactor